Jenkinsfile

```diff
@@ -10,14 +10,21 @@ def buildAndDeployDockerImage() {
     version = sh (script: "date '+%Y%m%d.%H%M%S'", returnStdout: true)
 
     docker.withRegistry("https://eu.gcr.io", "gcr:${gcpProject}") {
+
         sh """ docker build \
-            --build-arg REPO=${gcpProject} \
             --build-arg VERSION="${version}" \
-            --build-arg GIT_SHA="${gitSha}" \
+            --build-arg GIT_SHA="${gitSha}" . \
             --build-arg BIN=solr-operator \
+            -t solr-operator-build \
+            -f ./build/Dockerfile.build
+        """
+
+        sh """ docker build \
+            --build-arg BUILD_IMG=solr-operator-build . \
             -t ${imageName}:${buildTag} \
-            -f build/Dockerfile.build .
+            -f ./build/Dockerfile.slim
```

Comment:
Oh just noticed, you need ` .` on the end of both of these docker commands

Reply:
The dots are there. It's just further up. Want me to put it on the end for readability?

Reply:
Oh that's fine, I thought the dot had to be on the end